resources/simple-web-app/set_env_var.yaml

```diff
@@ -0,0 +1,4 @@
+path_to_image: resources/simple-web-app/lonely_feynman.tar
+ports: 3333:4444
+env: ["TEST_FOO=This is a test"]
```

Comment:
Also consider the test that checks that at least 2 of them setted, not only first one. 

for example, consider testing
 url ....`?prop=name` should return `Example Developer`
 url ....`?prop= job` should return `Developer`